plasma_framework/contracts/src/exits/interfaces/IStateTransitionVerifier.sol

```diff
@@ -9,7 +9,7 @@ interface IStateTransitionVerifier {
     function isCorrectStateTransition(
         bytes calldata inFlightTx,
         bytes[] calldata inputTxs,
-        uint256[] calldata inputUtxosPos
+        uint16[] calldata outputIndexOfInputTxs
```

Comment:
Could you add a doc for this function? `@param`

Reply:
okay, I add the doc as following:
I renamed `inFlightTx` to `txBytes` as from this verifier's point of view, it does not care that transaction is in-flight exiting or not.
```
/**
    * @notice Verifies state transition logic.
    * @param txBytes the targeting transaction that is doing the state transition verification
    * @param inputTxs input transactions of the targeting transaction checking state transition
    * @param outputIndexOfInputTxs the output index of the input txs that the transaction input is pointing to
    */
    function isCorrectStateTransition(
        bytes calldata txBytes,
        bytes[] calldata inputTxs,
        uint16[] calldata outputIndexOfInputTxs
    )
        external
        view
        returns (bool);
```